Tuxedo.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
```

Comment:
Is it worthwhile to explain what you're requiring here? Essentially adding comments for this uncommented file.